### PR TITLE
Derive type restrictions for derived displays from their source UI.

### PR DESCRIPTION
**Pull request by leftclickben**
+ Modify XSLT.
+ Fix formatting in XSLT (unrelated).
+ Update profile XML file with the result (some minor manual tweaks relating to CDATA sections).

### CODE REVIEW

File: profile/rwahs.xml [hunk 1]
```diff
@@ -9097,6 +9109,9 @@
           <name>Subject List display</name>
         </label>
       </labels>
+      <typeRestrictions>
+        <restriction type="Subject"/>
```
Comment (leftclickben) [inline profile/rwahs.xml]: Is this right?  It corresponds to the UI but is the value actually correct?

File: profile/rwahs.xml [hunk 2]
```diff
@@ -9137,6 +9152,9 @@
           <name>Conservation display</name>
         </label>
       </labels>
+      <typeRestrictions>
+        <restriction type="Conservation"/>
```
Comment (leftclickben) [inline profile/rwahs.xml]: As above, is this right?

File: profile/rwahs.xml (hunk 1 above)
Comment (kehh) [inline profile/rwahs.xml]: Yes, don't ask me why.

File: profile/rwahs.xml (hunk 2 above)
Comment (kehh) [inline profile/rwahs.xml]: si

**Pull request merged**